### PR TITLE
Navigationbar color

### PR DESCRIPTION
**Pull request by janga1997**
Fixes the blue color of the navigation on the calendar widget to grey, from blue.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+2.0.6 (2016-05-1)
+------------------
+
+Fixes:
+
+-Changed the color of the navigation in the calendar widget to grey(from blue) using inherit.
```
Comment (thet) [inline CHANGES.rst]: Missing space between ``-`` and ``Changed`` makes this sentence look bold.
Please add a space.

Comment (thet) [inline CHANGES.rst]: Also: Please add your changelog entry to the 2.0.9 section, which is still unreleased.

File: plone/app/event/browser/resources/event.css [hunk 2]
```diff
@@ -169,6 +169,10 @@ article.vevent a.event_ical img {
     margin: 0 -.5em;
 }
 
+.portletCalendar a.calendarNext,.portletCalendar a.calendarPrevious{
+    color:inherit;
```
Comment (thet) [inline plone/app/event/browser/resources/event.css]: Nice fix!

**Comment by thet** [comment]
Thanks for the fix!
Looks like, you are not part of the Github plone organisation, which is a hint that you haven't signed the Plone Contributor Agreement: http://docs.plone.org/develop/coredev/docs/agreement.html
You need to sign it, otherwise we cannot merge your PR.

**Comment by janga1997** [comment]
@thet I have signed the Contributor Agreement. I guess the tag of Plone foundation Member signifies that?

File: CHANGES.rst [hunk 3]
```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+2.0.9 (2016-05-1)
+------------------
```
Comment (gforcada) [inline CHANGES.rst]: Please fix the changelog header, you need to add your fix description under the ``2.0.9 (unreleased)`` header

**Comment by gforcada** [comment]
@janga1997 sorry but that was not what I meant. I meant this: https://github.com/plone/plone.app.event/commit/a5e4951b98b355beaf3a05abaae78a673a4d413a

@jensens a review-before-merging is always good ;-)

**Comment by jensens** [comment]
@gforcada i did, but not carefully enough. sorry. 
@janga1997 may you please fix this?

**Comment by gforcada** [comment]
@jensens I already fixed on master, that's the commit I pointed him to :-)

**Comment by janga1997** [comment]
@gforcada Thanks for the help!
 I am new at this, and help like yours is always appreciated.

**Pull request merged**